ui/src/app/kvm/views/KVMDetails/KVMResources/NumaResources/NumaResourcesCard/NumaResourcesCard.tsx

```diff
@@ -89,6 +90,8 @@ const NumaResourcesCard = ({ numaId, podId }: Props): JSX.Element => {
               allocated: numa.cores.allocated.length,
               free: numa.cores.free.length,
             }}
+            pinned={numa.cores.allocated}
+            available={available}
```

Comment:
Did you want to show the free cores per numa node? `numa.cores.free`? Otherwise it shows all the free cores on the whole pod object.

Reply:
Oh right, this is so much easier as a solution, what was I thinking!